Vungle/VungleBannerCustomEvent.m

```diff
@@ -29,33 +29,32 @@ @interface VungleBannerCustomEvent () <VungleRouterDelegate>
 
 @implementation VungleBannerCustomEvent
 
-- (void)requestAdWithSize:(CGSize)size customEventInfo:(NSDictionary *)info {
+- (void)requestAdWithSize:(CGSize)size customEventInfo:(NSDictionary *)info adMarkup:(NSString *)adMarkup {
     self.placementId = [info objectForKey:kVunglePlacementIdKey];
     self.options = nil;
     self.bannerSize = size;
     self.bannerInfo = info;
     self.isAdCached = NO;
-
+    
     self.timeOutTimer = [NSTimer scheduledTimerWithTimeInterval:BANNER_TIMEOUT_INTERVAL repeats:NO block:^(NSTimer * _Nonnull timer) {
         if (!self.isAdCached) {
             [[VungleRouter sharedRouter] clearDelegateForRequestingBanner];
         }
     }];
-
+    
     [[VungleRouter sharedRouter] requestBannerAdWithCustomEventInfo:info size:size delegate:self];
 }
 
-- (void)requestAdWithSize:(CGSize)size customEventInfo:(NSDictionary *)info adMarkup:(NSString *)adMarkup {
-    // Need to verify that we don't need to send the adMarkup string
-    [self requestAdWithSize:size customEventInfo:info];
-
-}
-
 - (void)dealloc {
     // call router event to transmit close to SDK for report ad finalization / clean up
 //    [[VungleRouter sharedRouter] completeBannerAdViewForPlacementID:self.placementId];
 }
 
+- (BOOL)enableAutomaticImpressionAndClickTracking
```

Comment:
According to Dravya Nataraj’s comments “It would be better to disable auto tracking if we can track them in the adapters at the closest possible callbacks”. So I disable automatic tracking. :)

Reply:
> > @YueVungle thanks for sharing your changes. I've added two comments - one concerning the Sample App that is used by QA and how the `localExtras` object will be updated from that application to pass in play options / settings, and one concerning the `trackClick` method. Please let me know if you have any other follow up questions here. Thanks!
> 
> Hi Clarke, I have checked VungleInterstitialCustomEvent class, it also uses NSUserDefaults to get parameters to pass in play options / settings, which could be replaced by localExtras, do we need to modify it in this version or in 6.5.x version? Thanks so much. ^_^

@YueVungle thanks for following up on this - I believe we should use the same approach in both places - please use the `localExtras` approach in `VungleInterstitialCustomEvent` class as well (I've also commented in the test app PR regarding this). Thanks!

Reply:
Based on a conversation between @shiv-vungle and Dravya, we will not be implementing `trackClick` (mentioned below). As such, we should not implement `trackImpression` and keep the automatic impression and click tracking enabled for now. Matt Bennion says that proper click tracking support is on the roadmap, so we'll be able to revisit this at some point in the future.

Reply:
OK, remove override enableAutomaticImpressionAndClickTracking method. :)